src/snowflake/snowpark/_internal/proto/update-from-local.sh

```diff
@@ -23,9 +23,9 @@ pushd $SRC_ROOT/Snowpark
 popd
 
```

Comment:
Thanks for fixing those

Reply:
easiest may be actually to set $SCRIPT_DIR=$SCRIPT_DIR/generated/

Reply:
updated!